src/main/webapp/app/exercises/shared/result/feedback-collapse.component.html

```diff
@@ -5,7 +5,8 @@
 <ng-template #longText>
     <div *ngIf="isCollapsed; else opened">
         <fa-icon class="fa-angle" [icon]="'angle-right'" (click)="isCollapsed = !isCollapsed"></fa-icon>
-        {{ text.substring(0, 100) }} ...
+        <!-- Only show first line as preview -->
+        {{ text.split('\n', 1)[0] }} ...
```

Comment:
```suggestion
        {{ text.substr(0, str.indexOf("\n") }} ...
```
Should be much more efficient and in case there is no new line, it returns an empty string.